deploy-monitoring.sh

```diff
@@ -1,28 +1,36 @@
 #!/bin/bash
 
 # Change the namespace to that of your project
-NAMESPACE=myproject
+NAMESPACE=${1:-myproject}
```

Comment:
@fgiorgetti  that allows you to either specify the NS as a parameter or change search/replace myproject across all repo, right?

Reply:
@pwright this is the only place where you need to change it.